sdk/core/azure-core/inc/http/bodyStream.hpp

```diff
@@ -0,0 +1,269 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#ifdef Posix
+#include <unistd.h>
+#endif
+
+#ifdef Windows
+#define WIN32_LEAN_AND_MEAN
+#define NOMINMAX
+#include <Windows.h>
+#endif // Windows
```

Comment:
Nit: is it a good idea to include `unistd.h` and `Windows.h` in header file? It'll pollute the global namespace.

Reply:
Obviously, we prefer not to do this. For linux we can problem get rid of it because a file descriptor is an 'int'.
For Windows, the HANDLE is the data type. Maybe we can get rid of this header file with a small hack and define the FileBodyStream ctor parameter as a uint32_t. We'll just have to add some comment/documentation.